### PR TITLE
`shannon_control` cleanups

### PR DESCRIPTION
**Pull request by vinayaknagpal**
Added functionality for `shannon_control` to manage ZPU configurations using human readable `.pt` files. The `.pt` files are created in addition to `.pb` versions. I wasn't sure if things in ZenCode would break if we got rid of `.pb` altogether. Eventually it would be good to make sure that all configurations are read from the proper C++ API call and then we can get rid of the opaque `.pb` configurations for good. 

Minor cleanups that improve logging, more understandable method names and sample files in `shannon_command/commands` that help to test functionality. 

This PR also includes "soft" removal of the broadcast service from `shannon_bus_relay`

### CODE REVIEW

File: cpp/app/shannon_control/src/filesystem.cc [hunk 1]
```diff
@@ -33,8 +38,9 @@ ControlFileSystem::ControlFileSystem(std::string root)
 
   for (auto& path: paths) {
     if (!fs::exists(path))
-      CHECK(fs::create_directories(path));
+      CHECK(fs::create_directories(path)) << "Failed to create control filesystem directories.";
   }
+  LOG(INFO) << "Control filesystem directories created at " << root << "\n";
```
Comment (ghsyu) [inline cpp/app/shannon_control/src/filesystem.cc:43]: This might be better at DEBUG level

Comment (vinayaknagpal) [reply cpp/app/shannon_control/src/filesystem.cc:43]: Apparently google log doesn't have a DEBUG level. Its INFO, WARNING, ERROR, FATAL

File: cpp/app/shannon_control/src/filesystem.h [hunk 2]
```diff
@@ -5,6 +5,7 @@
 #include <vector>
 
 #include "shannon/protocol.h"
+#include "shannon/core/cxx/protocol.h"
```
[the remark this answers is not in the document]
Comment (vinayaknagpal) [reply cpp/app/shannon_control/src/filesystem.h:8]: `shannon::SerializeToString` is defined there thats why I had to add it.

File: cpp/app/shannon_control/src/filesystem.cc (hunk 1 above)
Comment (ghsyu) [reply cpp/app/shannon_control/src/filesystem.cc:43]: that's ok then

File: cpp/app/shannon_control/src/filesystem.h (hunk 2 above)
[the remark this answers is not in the document]
Comment (ghsyu) [reply cpp/app/shannon_control/src/filesystem.h:8]: ok, then you should specify which protocol.h on line 54

**Pull request merged**